.github/copilot-instructions.md

```
@@ -647,6 +647,65 @@ The `Qwiq.Package.Tests` project validates NuGet package contents using Verify.
 - Compare package manifests and contents against verified baselines
 - Will fail if run without first creating packages
 
+**⚠️ CRITICAL: When NuGet Package Contents Change**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)